InsertionChangeLogGenerator/ChangeLogGenerator.cs

```diff
@@ -185,6 +187,55 @@ static string GetHrefOrEmpty(Tuple<int, string> url)
             }
         }
 
+        public static void SaveAsMarkdown(IList<Commit> commits, string path)
+        {
+            var resultMarkdownPath = Path.Combine(path, "results.md");
+            Directory.CreateDirectory(Path.GetDirectoryName(resultMarkdownPath));
+
+            if (File.Exists(resultMarkdownPath))
+            {
+                File.Delete(resultMarkdownPath);
+            }
+
+            using (var w = File.AppendText(resultMarkdownPath))
+            {
+                w.WriteLine(
+                @"|Pull Request |Issue(s) |Commit |Author |Commit Message |");
```

Comment:
This is missing the `Area` from the html equivalent. 

Is that on purpose? 

Area is used for M2/QB scenarios to add more details.

Reply:
I just saw the teams discussion. 

Seems like it's on purpose?

Reply:
> Seems like it's on purpose?

Yep, we don't need to write the Area on GitHub for any reason I'm aware of.

In case it's not clear, this does not remove anything from the HTML file.